README.md

```diff
@@ -1,2 +1,2 @@
 # bpmn-js
-Javascript libs to render and model BPMN diagrams
+Process Analytics: An embeddable solution that delivers interactive data visualization of process execution for monitoring and reporting.
```

Comment:
I though **Process Analytics** was the commercial project.

Does this library just display BPMN diagram with extensions (like annotations, colors, ...) ? So, no need to talk about monitoring & reporting here.